tests/test_numpy_dtypes.cpp

```diff
@@ -259,7 +259,26 @@ TEST_SUBMODULE(numpy_dtypes, m) {
     catch (...) { return; }
 
     // typeinfo may be registered before the dtype descriptor for scalar casts to work...
-    py::class_<SimpleStruct>(m, "SimpleStruct");
+    py::class_<SimpleStruct>(m, "SimpleStruct")
+        // Explicit construct with braces to ensure zero-valued initialization.
+        .def(py::init([]() { return SimpleStruct{}; }))
```

Comment:
This doesn't work on GCC 4.8 (manylinux1)

Reply:
Haven't tested this, but I think this should work:
```suggestion
        .def(py::init([]() {
            SimpleStruct s = {0};
            return s;
         }))
```
Alternatively, the ` = {0}` part can be skipped, leaving just `SimpleStruct s;`